src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -286,9 +286,15 @@ export const ExpressCheckoutUI = ({ order }: ExpressCheckoutUIProps) => {
     shippingAddress,
     expressPaymentType,
     shippingRate,
-  }: StripeExpressCheckoutElementConfirmEvent) => {
+  }: StripeExpressCheckoutElementConfirmEvent & {
+    expressPaymentType: "apple_pay" | "google_pay"
+  }) => {
```

Comment:
I believe we already have the type of `expressPaymentType` in `StripeExpressCheckoutElementConfirmEvent` already 👀